client/src/components/About/about.js

```diff
@@ -1,81 +1,126 @@
 import React, { Component } from "react";
+import "./about.css";
+import picture from "../img/piggy_gg.png";
```

Comment:
what type of picture is this? this isnt very descriptive in the variable name. there are like 1289435u9y2354892358233258932765823 pictures in the world. which one is it? piggy_gg.png i dont know what picture this means is it a logo? a small piggy picture? a big piggy picture?